avocado/plugins/human.py

```diff
@@ -59,8 +60,11 @@ def start_test(self, result, state):
         else:
             name = "<unknown>"
             uid = '?'
-        LOG_UI.debug(' (%s/%s) %s:  ', uid, result.tests_total, name,
-                     extra={"skip_newline": True})
+        if self.runner == 'nrunner':
+            LOG_UI.debug(' (%s/%s) %s: STARTED', uid, result.tests_total, name)
```

Comment:
It will be interesting to see what the parallel suggested output format will end up being. Hopefully it manages to remain as compact as the serial output format and yet as informative.

Reply:
It's not perfect at this point, because it will give feedback when tests start, and then when they end.  The better approach would be to keep track of the tests running, redraw the screen, and reuse the same line. That, or something even better (@beraldoleal "fancy dots") is certainly on the radar.

Reply:
Yes, perhaps we could integrate this in the human output plugin without destroying the good enough format for the serial runner version.